misc/scripts/models-as-data/generate_flow_model.py

```diff
@@ -133,6 +133,9 @@ def readRows(self, bqrsFile):
 
     def asCsvModel(self, superclass, kind, rows):
         classTemplate = """
+/**
+ * THIS FILE IS AN AUTO-GENERATED MODELS AS DATA FILE. DO NOT EDIT.
+ */
```

Comment:
This should probably go before the `/** Definitions of taint steps in the {self.shortname} framework */` comment.

Reply:
Yes, you are right. No reason to have it there for each class.